package.json

```diff
@@ -55,6 +56,9 @@
     "transform": {
       "\\.(ts)$": "<rootDir>/node_modules/ts-jest/preprocessor.js"
     },
-    "testRegex": "/src/.*\\.spec.(ts|js)$"
+    "testRegex": "/src/.*\\.spec.(ts|js)$",
+    "collectCoverageFrom": [
+      "src/*.{ts,js}"
```

Comment:
Will this include all the spec files and examples themself? I'd rather only have the source files.